Modules/Core/src/Controllers/Gateways/PaypalController.php

```
@@ -24,11 +26,11 @@ public function __construct()
     public function paypalCreateOrder($invoice_url_key)
     {
         // Check if the invoice exists and is billable
-        $this->load->model('invoices/mdl_invoices');
+// TODO: Use dependency injection - $this->load->model('invoices/mdl_invoices');
         $invoice = $this->mdl_invoices->where('ip_invoices.invoice_url_key', $invoice_url_key)->get()->row();
         // Check if the invoice is payable
         if ($invoice->invoice_balance <= 0) {
-            $this->session->set_userdata('alert_error', lang('invoice_already_paid'));
+            session(['alert_error', lang('invoice_already_paid'));
```

PHPStan (2.1.31)

Syntax error, unexpected ')', expecting ']' on line 33


(phpstan.parse)